src/app/components/Footer/index.js

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+// assets
+import logoImage from 'assets/images/darkerLogo.png';
+
+import LinkField from './LinkField';
+
+const Containter = styled.div`
+  bottom: 0;
```

Comment:
when we set `bottom`, `top`, `left`, `right`, we have to control our attribute `position`. In this case, can you add `position: relative` inside this style?